ReactAndroid/src/androidTest/js/ScrollViewTestModule.js

```diff
@@ -37,54 +36,52 @@ class Item extends React.Component {
   }
 }
 
-var getInitialState = function() {
-  var data = [];
-  for (var i = 0; i < NUM_ITEMS; i++) {
-    data[i] = {text: 'Item ' + i + '!'};
+class ScrollViewTestApp extends React.Component {
+  constructor() {
+    super();
+    this.scrollView = React.createRef();
   }
-  return {
-    data: data,
-  };
-};
 
-var onScroll = function(e) {
-  ScrollListener.onScroll(
-    e.nativeEvent.contentOffset.x,
-    e.nativeEvent.contentOffset.y,
-  );
-};
+  getInitialState = () => {
+    var data = [];
+    for (var i = 0; i < NUM_ITEMS; i++) {
+      data[i] = {text: 'Item ' + i + '!'};
+    }
+    return {
+      data: data,
+    };
+  };
 
-var onScrollBeginDrag = function(e) {
-  ScrollListener.onScrollBeginDrag(
-    e.nativeEvent.contentOffset.x,
-    e.nativeEvent.contentOffset.y,
-  );
-};
+  onScroll = e => {
+    ScrollListener.onScroll(
+      e.nativeEvent.contentOffset.x,
+      e.nativeEvent.contentOffset.y,
+    );
+  };
```

Comment:
All functions need to meet two conditions before you should consider binding them:

1. You're using `this` inside the function.
2. You're passing the function by reference to component's prop or as a callback to some function. 

Since we're not using `this` inside `onScroll`, we don't need to bind this function. In fact, since we're not using `this`, we could leave `onScroll` as is (i.e: Implemented outside the `ScrollViewTestApp` class). 

Okay, lets just move this back to where it was before.